newIDE/app/src/EventsSheet/index.js

```diff
@@ -1105,85 +1130,96 @@ export default class EventsSheet extends React.Component<Props, State> {
                           ref={eventContextMenu =>
                             (this.eventContextMenu = eventContextMenu)
                           }
-                          buildMenuTemplate={() => [
-                            {
-                              label: 'Copy',
-                              click: () => this.copySelection(),
-                              accelerator: 'CmdOrCtrl+C',
-                            },
-                            {
-                              label: 'Cut',
-                              click: () => this.cutSelection(),
-                              accelerator: 'CmdOrCtrl+X',
-                            },
-                            {
-                              label: 'Paste',
-                              click: () => this.pasteEvents(),
-                              enabled: hasClipboardEvents(),
-                              accelerator: 'CmdOrCtrl+V',
-                            },
-                            {
-                              label: 'Delete',
-                              click: () => this.deleteSelection(),
-                              accelerator: 'Delete',
-                            },
-                            {
-                              label: 'Toggle disabled',
-                              click: () => this.toggleDisabled(),
-                              enabled: this._selectionCanToggleDisabled(),
-                            },
-                            { type: 'separator' },
-                            {
-                              label: 'Add New Event Below',
-                              click: () =>
-                                this.addNewEvent(
-                                  'BuiltinCommonInstructions::Standard'
+                          buildMenuTemplate={() => {
```

Comment:
Thanks for the changes :) Now you should be able to go back to a function directly returning an array: 
```
() => [
   ...
```

Slightly shorter/less indentation :)

Reply:
Done ✅